elements/campaign-display/campaign-display.scss

```diff
@@ -1,8 +1,24 @@
 campaign-display {
-  display: block;
-
+  color: rgba(112, 112, 112, 1);
+  display: inline-block;
+  font-family: 'Helvetica', Arial, sans-serif;
+  font-size: 10px;
+  text-transform: uppercase;
+  .inner a {
```

Comment:
Want to note, `.inner` seems like too generic a class name, something else might style this on accident. But that change is much bigger than this PR and includes `omni`.